themes/startwords/README.md

```diff
@@ -1,31 +1,28 @@
-# Blank
+# <div align="center"><img src="https://startwords.cdh.princeton.edu/logotype.svg" /> Theme</div>
 
-Blank — starter [Hugo](https://gohugo.io/) theme for developers. Use it to make your own theme.
+[![License: CC BY-NC-SA 4.0](https://img.shields.io/badge/License-CC%20BY--NC--SA%204.0-lightgrey.svg)](https://creativecommons.org/licenses/by-nc-sa/4.0/)
```

Comment:
The CC license will be for the content, but not the theme/code; I think @thatbudakguy suggested Apache 2 license, which we're using for our other code.